Task10/Task10/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace Task10
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите два натуральных числа от 5 до 20");
+            var m = int.Parse(Console.ReadLine());
+            var n = int.Parse(Console.ReadLine());
+            int[,] natNum = new int[m, n];
+            Random rand = new Random();
+
+            for (int i = 0; i < m; i++)
+            {
+                for (int j = 0; j < n; j++)
+                {
+                    int randomNum = rand.Next(0, 99);
+                    natNum[i, j] = randomNum;
+                }
+            }
+            PrintTable(natNum);
+
+            Console.Write("Введите натуральное число ");
+            var k = int.Parse(Console.ReadLine());
+            First(natNum, k);
+
+            Second(natNum);
+
+            Console.ReadKey();
+        }
+        static void PrintTable(int[,] arr)
+        {
+            for (int i = 0; i < arr.GetLength(0); i++)
+            {
+                for (int j = 0; j < arr.GetLength(1); j++)
+                    Console.Write($"{arr[i, j]} ");
```

Comment:
Чтобы табличка была красивой, лучше так
```suggestion
                    Console.Write($"{arr[i, j], 2} ");
```